From: Ben Hutchings <ben@decadent.org.uk>
Subject: cirrus: Disable by default
Forwarded: not-needed

This kernel driver is incompatible with older versions of
xserver-xorg-video-cirrus, so do not enable it by default.  It will be
enabled by a modprobe configuration file installed by
xserver-xorg-video-modesetting.

--- a/drivers/gpu/drm/cirrus/cirrus_drv.c
+++ b/drivers/gpu/drm/cirrus/cirrus_drv.c
@@ -15,7 +15,7 @@
 
 #include "cirrus_drv.h"
 
-int cirrus_modeset = -1;
+int cirrus_modeset = 0;
 
 MODULE_PARM_DESC(modeset, "Disable/Enable modesetting");
 module_param_named(modeset, cirrus_modeset, int, 0400);
